test/tools/unified-spec-runner/entities.ts

```diff
@@ -373,14 +375,14 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
         const client = map.getEntity('client', entity.database.client);
         const db = client.db(
           entity.database.databaseName,
-          patchDbOptions(entity.database.databaseOptions)
+          patchDbOptions(entity.database.databaseOptions!)
         );
         map.set(entity.database.id, db);
       } else if ('collection' in entity) {
         const db = map.getEntity('db', entity.collection.database);
         const collection = db.collection(
           entity.collection.collectionName,
-          patchCollectionOptions(entity.collection.collectionOptions)
+          patchCollectionOptions(entity.collection.collectionOptions!)
```

Comment:
Nit, instead of non-null, we can add optionality to the input to these patch functions since options are also optional to our driver APIs, like `db.collection(entity.collection.collectionName, undefined)` is legal

Reply:
sure